apps/cli/src/components/chat/terminal-chat.tsx

```diff
@@ -53,7 +53,7 @@ export function TerminalChat({
   useEffect(() => {
     if (!historyMessages?.length || hasLoadedHistory) return;
 
-    const messages = convertEventToMessages(historyMessages);
+    const messages = convertEventToMessages(historyMessages, true);
```

Comment:
```suggestion
    const messages = convertEventToMessages({events: historyMessages, isHistory: true});
```

Could we change this into an object so its easier to understand what's being passed here? its a bit "magical" passing a true value

Reply:
sure